src/Interpreters/Streaming/Aggregator.cpp

```diff
@@ -650,12 +770,43 @@ AggregatedDataVariants::Type Aggregator::chooseAggregationMethodTimeBucketTwoLev
 }
 /// proton: ends
 
-template <bool skip_compiled_aggregate_functions>
+template <bool use_compiled_functions, bool skip_expanded_data>
 void Aggregator::createAggregateStates(AggregateDataPtr & aggregate_data) const
 {
+    /// Initialize reserved UpdatedDataEx
+    assert(aggregate_data);
+    if constexpr (!skip_expanded_data)
+    {
+        if (expanded_data_type == ExpandedDataType::UpdatedWithRetracted)
+            new (aggregate_data) RetractedDataEx();
+        else if (expanded_data_type == ExpandedDataType::Updated)
+            new (aggregate_data) UpdatedDataEx();
+    }
+
+    if constexpr (use_compiled_functions)
```

Comment:
Let's have a comment or comment it out for compiled function since it doesn't work for streaming processing in general for now ? We can't GC the data etc properly

Reply:
in fact, we have supported it, for some support compile aggr functions, for example `min/max/count .etc` , and i think they can GC correctly

Reply:
we can not support it first, and decide whether to enable JIT after performing separate perf verification later for streaming query.